packages-exp/functions-exp/src/api.ts

```diff
@@ -55,6 +56,23 @@ export function getFunctions(
   return functionsInstance;
 }
 
+/**
+ * Modify this instance to communicate with the Cloudd Functions emulator.
```

Comment:
typo on "Cloudd", also in `functions-types/index.d.ts` and `firebase/index.d.ts` and both `service.ts` files.